src/Internal.AspNetCore.Sdk/build/Common.targets

```diff
@@ -10,16 +10,72 @@ For single-tfm projects, this will be imported from build/Internal.AspNetCore.Sd
+    <PackDependsOn Condition="'$(NuspecFile)' != ''">$(PackDependsOn);_GenerateSymbolsNupkg</PackDependsOn>
+
+    <!-- workaround https://github.com/NuGet/Home/issues/4726 -->
+    <IncludeSymbols Condition="'$(NuspecFile)'!=''">false</IncludeSymbols>
+    <IncludeSource Condition="'$(NuspecFile)'!=''">false</IncludeSource>
+
     <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);_EnsureDebugTypeIsPortable</TargetsForTfmSpecificBuildOutput>
   </PropertyGroup>
 
+  <Target Name="_GenerateSymbolsNupkg" >
+    <Error Condition="'$(NuspecFile)' != '' AND '$(SymbolsNuspecFile)' == ''"
+      Text="NuspecFile was specified but SymbolsNuspecFile was not specified."
+      Code="KRB5006" />
+
+    <ConvertToAbsolutePath Paths="$(SymbolsNuspecFile)">
+      <Output TaskParameter="AbsolutePaths" PropertyName="SymbolNuspecFileAbsolutePath" />
+    </ConvertToAbsolutePath>
+
+    <ConvertToAbsolutePath Paths="$(PackageOutputPath)symbols-nupkg\">
+      <Output TaskParameter="AbsolutePaths" PropertyName="SymbolsPackageOutputAbsolutePath" />
+    </ConvertToAbsolutePath>
+
+    <PackTask PackItem="$(PackProjectInputFile)"
+              PackageVersion="$(PackageVersion)"
+              PackageId="$(PackageId)"
+              Title="$(Title)"
+              Authors="$(Authors)"
+              Description="$(PackageDescription)"
+              Copyright="$(Copyright)"
+              RequireLicenseAcceptance="$(PackageRequireLicenseAcceptance)"
+              LicenseUrl="$(PackageLicenseUrl)"
+              ProjectUrl="$(PackageProjectUrl)"
+              IconUrl="$(PackageIconUrl)"
+              ReleaseNotes="$(PackageReleaseNotes)"
+              Tags="$(PackageTags)"
+              AssemblyName="$(AssemblyName)"
+              PackageOutputPath="$(SymbolsPackageOutputAbsolutePath)"
+              IncludeSymbols="false"
+              IncludeSource="false"
+              PackageTypes="$(PackageType)"
+              IsTool="$(IsTool)"
+              RepositoryUrl="$(RepositoryUrl)"
+              RepositoryType="$(RepositoryType)"
+              NoPackageAnalysis="$(NoPackageAnalysis)"
+              MinClientVersion="$(MinClientVersion)"
+              Serviceable="$(Serviceable)"
+              ContinuePackingAfterGeneratingNuspec="$(ContinuePackingAfterGeneratingNuspec)"
+              NuspecOutputPath="$(SymbolNuspecFileAbsolutePath)"
+              IncludeBuildOutput="false"
+              RestoreOutputPath="$(RestoreOutputAbsolutePath)"
+              NuspecFile="$(SymbolNuspecFileAbsolutePath)"
+              NuspecBasePath="$(NuspecBasePath)"
+              NuspecProperties="$(NuspecProperties)"
+              AllowedOutputExtensionsInPackageBuildOutputFolder="$(AllowedOutputExtensionsInPackageBuildOutputFolder)"
+              AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder="$(AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder)"/>
+
+    <Copy
```

Comment:
Maybe use `<Move>` instead? This task doesn't use incrementalism, so there doesn't appear to be an advantage to `<Copy>`.